src/body/boxed.rs

```diff
@@ -11,13 +11,20 @@ pub struct Boxed {
     body: Pin<Box<dyn Body<Data = Bytes, Error = Error> + Send>>,
 }
 
+/// Maps the error type of a body to [Error].
+#[must_use = "streams do nothing unless polled"]
+struct MapError<B> {
```

Comment:
While this adds a small amount of overhead to a Boxed body, it is something that the caller would have to do anyways. An alternative would be using `Box<dyn StdError + Send + Sync>` that may be worth exploring before the first release. Until then, this sufficient and allows for any type that implements `Body<Data = Bytes>` to be used. Here is an updated version of the hello-world example that uses a String in combination with Boxed for the response body.

```rust
use via::body::Boxed;
use via::{Next, Request, Response, Result};

async fn hello(request: Request, _: Next) -> Result<Response> {
    // Get a reference to the path parameter `name` from the request uri.
    let name = request.param("name").required()?;

    Response::build()
        .body(Boxed::new(format!("Hello, {}!", name)))
        .finish()
}
```